rpmbuild/main.py

```diff
@@ -225,6 +226,9 @@ def build_srpm(args, config):
     log.info("Output: {0}".format(
         os.listdir(resultdir)))
 
+    # extract spec file from SRPM
+    subprocess.run(f"rpm2archive -n < {locate_srpm(resultdir)} | tar xf - '*.spec'", shell=True, check=False)
```

Comment:
Here we should decide what happens if there are multiple source RPM files.

Reply:
could there be a situation where there are more SRPMs in the resultdir?

Reply:
It's not a valid case, but anything can happen (we've seen this already).